tuned/ppd/config.py

```diff
@@ -141,4 +141,4 @@ def load_from_file(self, config_file):
         self._ppd_to_tuned = ProfileMap(profile_dict_ac, profile_dict_dc)
         self._tuned_to_ppd = ProfileMap({v: k for k, v in profile_dict_ac.items()}, {v: k for k, v in profile_dict_dc.items()})
 
-        self._thinkpad_function_keys = cfg.getboolean(MAIN_SECTION, THINKPAD_FUNCTION_KEYS_OPTION, fallback=False)
+        self._sysfs_acpi_monitor = cfg.getboolean(MAIN_SECTION, SYSFS_ACPI_MONITOR_OPTION, fallback=False)
```

Comment:
```suggestion
        self._sysfs_acpi_monitor = cfg.getboolean(MAIN_SECTION, SYSFS_ACPI_MONITOR_OPTION, fallback=True)
```

Reply:
Please update to default to True even if the option is not explicitly defined.

Reply:
Sorry - I might have got the work flow wrong....I'm used to doing kernel commits, I don't play in user space often.

Could I just have accepted the commit suggestion above without having to make the change in my tree? Hope the below works and I haven't messed things up.

Reply:
Both approaches work fine :)